graph-subscriptions-api/README.md

```diff
@@ -0,0 +1,18 @@
+# Graph Subscriptions API
```

Comment:
I'm assuming every gateway operator would be running this component alongside their gateway?

Reply:
If so, shouldn't this be part of the gateway?

Reply:
I don't think so, for a few reasons:
- Adding a public API like this to the gateway process would add an additional attack surface that might degrade the gateway processes' ability to serve queries. This serves a much lower priority service, that doesn't impact that of the gateway.
- It would also introduce a new data flow edge where the gateway is consuming the data it puts out. This would be fine, except that this service might rely on some sort of ETL in between, like a materialized view for daily aggregations.